spacy_llm/tasks/span/task.py

```diff
@@ -62,7 +63,7 @@ def __init__(
         self._description = description
 
         if self._prompt_examples:
-            self._prompt_examples = list(self._check_label_consistency(self))
+            self._prompt_examples = list(check_label_consistency(self))
```

Comment:
I think it's fine to use the original code, refering to the method in `self`, as this was defined on L61. (imagine we'd change the definition on L61 in a future refactor, then we might miss that we're still using the "old" one here)

Reply:
Done in https://github.com/explosion/spacy-llm/pull/306/commits/19d3660926260ae1f159c782add4caeaf83476db.